scripts/save-random-trip.js

```diff
@@ -58,7 +58,11 @@ const main = async () => {
   let sourceAccount
   if (!args.source_account) {
     const { data: accounts } = await client.query(
-      Q('io.cozy.accounts').where({ account_type: 'tracemob' })
+      Q('io.cozy.accounts').where({
+        account_type: {
+          $or: ['tracemob', 'openpath']
+        }
+      })
```

Comment:
should be a partialIndex ;)

Reply:
Nice catch, I also fixed the drop-account and drop-accounts scripts

Reply:
True, but not a big deal for tooling, and even less for small databases such as accounts, IMO ;)